force-app/main/default/objects/Tv_Serie__c/validationRules/Tv_Serie_Name_Unique_Rule.validationRule-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Tv_Serie_Name_Unique_Rule</fullName>
+    <active>true</active>
+    <description>Tv Serie must be unique</description>
+    <errorConditionFormula>LEN(VLOOKUP($ObjectType.Tv_Serie__c.Fields.Name,
+								$ObjectType.Tv_Serie__c.Fields.Name,
+								Name))&gt;0</errorConditionFormula>
+    <errorDisplayField>Name</errorDisplayField>
+    <errorMessage>Tv Serie of this Name already exist.</errorMessage>
```

Comment:
typo in 'exists'

Reply:
s++

Reply:
I also changed validation formula, because this blocked edit of other fields, example like description.